config/vufind/config.ini

```diff
@@ -2098,6 +2098,11 @@ related[] = "Similar"
 ; records). Default is false.
 ;alwaysDisplayIndexRecordInStaffView = false
 
+; This setting controls the display order of subject headings from MARC records.
+; If set to "record" (the default) it will retain the order of headings from the
+; MARC record; if set to "field", it will sort numerically by MARC tag.
+;marcSubjectHeadingsSort = "record"
```

Comment:
Since this is commented out, I had intentionally set it to "field" so that you could uncomment it to switch away from default behavior. You might want to change it back, but if you prefer it this way, I won't argue further. :-)

Reply:
Done